packages/ui/src/theme/css/component/StorageBrowser/storageBrowser.scss

```diff
@@ -146,6 +146,15 @@
       white-space: nowrap;
       align-items: center;
       justify-content: flex-start;
+
+      &-icon {
+        &--action-progress {
+          animation-name: amplify-loader-circular;
+          animation-timing-function: linear;
+          animation-iteration-count: infinite;
+          animation-duration: var(--amplify-components-loader-animation-duration);
+        }
+      }
```

Comment:
Info: does this need to be within any selector aside from `-text-data-cell`?

Reply:
Afaict, this was our only use case for this. I considered lifting it up and using wildcard but I think just keeping it simple is probably right for now